config/test.exs

```diff
@@ -13,7 +13,8 @@ config :amqpx,
     host: "rabbit",
     virtual_host: "/",
     heartbeat: 30,
-    connection_timeout: 10_000
+    connection_timeout: 10_000,
+    obfuscate_password: false
```

Comment:
Can we obfuscate it by default (`obfuscate_password: true`)?

Reply:
Yep, you're right! Magic happens here https://github.com/primait/amqpx/pull/71/files#diff-c9c326eb26b1bf4c293a8d715ba017fe70808a7381641697331dfbf74171c0d4R21

However we need `obfuscate_password: false` in `config/test.exs` otherwise tests fails in CI/CD